src/odemis/driver/andorcam2.py

```diff
@@ -828,21 +828,18 @@ def __init__(self, name, role, device, emgains=None, shutter_times=None,
             # 0 => shutter always auto
             # > 0 => shutter auto if exp time + readout > period, otherwise opened
             # big value => shutter always opened
-            if self.hasShutter(shutter_times is not None):
+            if caps.CameraType == AndorCapabilities.CAMERATYPE_IXONULTRA:
+                # Special case for iXon Ultra -> leave it open (with 0, 0) (cf p.77 hardware guide)
```

Comment:
`(cf p.77 hardware guide)` can you mention a specific section? That will be helpful in case we get a new hardware guide